src/components/pages/collectRecordFormPages/BenthicPhotoQuadratForm/BenthicPhotoQuadratForm.js

```diff
@@ -1,76 +1,76 @@
-import PropTypes from 'prop-types'
-import React, { useState, useEffect, useCallback, useReducer } from 'react'
 import { toast } from 'react-toastify'
 import { useParams } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import React, { useState, useReducer, useEffect, useMemo, useCallback } from 'react'
```

Comment:
BPQ using the new protocol - modelled it after benthic lit/pit however there are still some things that 100% sure on